src/main/scala/com/gu/newsletterlistcleanse/GetCleanseListLambda.scala

```diff
@@ -1,40 +1,42 @@
 package com.gu.newsletterlistcleanse
 
 import com.amazonaws.services.lambda.runtime.Context
-import org.slf4j.{ Logger, LoggerFactory }
-
-/**
- * This is compatible with aws' lambda JSON to POJO conversion.
- * You can test your lambda by sending it the following payload:
- * {"name": "Bob"}
- */
-class GetCleanseListLambdaInput() {
-  var name: String = _
-  def getName(): String = name
-  def setName(theName: String): Unit = name = theName
-}
+import com.gu.identity.model.EmailNewsletters
+import com.gu.newsletterlistcleanse.db.{Campaigns, CampaignsFromDB}
+import org.slf4j.{Logger, LoggerFactory}
+
+import scala.beans.BeanProperty
+
+
+case class GetCleanseListLambdaInput(
+  @BeanProperty
+  newslettersToProcess: List[String]
+)
 
 object GetCleanseListLambda {
 
   val logger: Logger = LoggerFactory.getLogger(this.getClass)
+  val campaigns: Campaigns = new CampaignsFromDB()
```

Comment:
This is the DataAccessObject for this lambda. I'm still on the fence about my naming, feel free to rename!